app/processors/audio_processor.rb

```diff
@@ -46,39 +46,39 @@ def google_transcribe(wav, opts={})
     GoogleSpeech::Transcriber.new(File.open(wav.path), symbolize_options(opts)).transcribe
   end
 
-  def waveformjson_audio
```

Comment:
why commenting this out? why not delete?

Reply:
agreed